### PR TITLE
Fix typos in comments

### PR DESCRIPTION
**Pull request by gesellix**
Signed-off-by: Tobias Gesellchen <tobias@gesellix.de>

### CODE REVIEW

File: types/network/network.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ type EndpointSettings struct {
 }
 
 // NetworkingConfig represents the container's networking configuration for each of its interfaces
-// Carries the networink configs specified in the `docker run` and `docker network connect` commands
+// Carries the networing configs specified in the `docker run` and `docker network connect` commands
```
Comment (thaJeztah) [inline types/network/network.go]: You missed a 'k' here; s/networing/networking/

**Comment by gesellix** [comment]
@thaJeztah oh my. I updated the commit, thanks!

**Comment by thaJeztah** [comment]
haha, no worries :+1: 

LGTM

**Comment by vdemeester** [comment]
LGTM 🐯

**Pull request merged**